Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/ManagePaywallButton.swift

```diff
@@ -82,16 +82,22 @@ struct ManagePaywallButton: View {
     }
 
     private func openURL(_ url: URL) {
```

Comment:
Is there a reason we don't use `@Environment(\.openURL) var openURL` instead of this private function?

Reply:
Ooooh that's cool, I want to use that now haha.